### PR TITLE
Update data model

### PR DESCRIPTION
**Pull request by youaresofunny**
Rename materialCollection to materialTypeCollection
Add new materialCollection

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,6 +233,22 @@ const banjarCollection = [
 ];
 
 const materialCollection = [
+   {
+	worker: {
```
Comment (nicksarafa) [inline README.md]: what does `worker` represent in this instance? we'll have to `i18n` the database modals

Comment (youaresofunny) [reply README.md]: It is from 04.0 screen (also employee = true in person model), should we call it employee?

**Comment by nicksarafa** [comment]
👍

**Comment by nicksarafa** [comment]
CI doesn't pass but it will following merge

**Comment by nicksarafa** [comment]
rebased onto `develop` to ensure the CI passes

**Pull request merged**